__tests__/delete-selected/1.0/delete-selected.test.js

```diff
@@ -0,0 +1,63 @@
+import deleteSelected from '../../../functions/delete-selected/1.0';
+
+// TO DO: Mock data to check if deleted records match return
```

Comment:
@jwielinga  Like I said, the tests are WIP and need to be mocked so this can all be ignored.

Reply:
@vdberganton no worries. Just want to give feedback so Gijs knows what he can work on